code/modules/mob/living/carbon/human/inventory.dm

```diff
@@ -171,7 +172,7 @@
 				dropItemToGround(belt)
 		w_uniform = null
 		update_suit_sensors()
-		if(!QDELETED(src)) 
+		if(!QDELETED(src))

```

Comment:
It's just a whitespace change; just ignore unless egregious
https://github.com/tgstation/tgstation/pull/29826/files?w=1